Description: Fix FTBFS with glew 2.1.0
 Recent glew introduces conflicting declaration with <GL/glext.h>
Author: Bertrand Marc <bmarc@debian.org>
Bug-Debian: http://bugs.debian.org/917510

--- a/src/include/pengine.h
+++ b/src/include/pengine.h
@@ -28,7 +28,6 @@
 #include <physfs.h>
 
 #include <GL/glew.h>
-#include <GL/glext.h>
 
 // TinyXML is built into the pengine library
 #include "tinyxml2.h"
